benchmarks/DSE.Open.Benchmarks/Observations/ObservationHashValueBenchmarks.cs

```diff
@@ -52,49 +51,49 @@ public ulong[] UsingGetRepeatableHashCode()
 
 public static class TestMeasures
 {
-    public static readonly Measure<bool> BinaryMeasure = new(
+    public static readonly Measure<Binary> BinaryMeasure = new(
```

Comment:
Can we think of a name slightly more specific than just "Binary"?

Reply:
Not really